### ISSUE TITLE
Turn off a screen using proximity sensor during navigation.

### ISSUE THREAD

**Issue by EvgenyZaretskiy** [osmandapp/OsmAnd#13109]
Hello!

Now we have a feature to turn on a screen during navigation using the proximity sensor. Is it possible to add a feature to turn OFF a screen the same way during navigation?

Problem: driving a bike in rainy condition using the waterproof box, can't charge a smartphone and wanna have the screen turned off most of the time. OK. Can turn on the screen bringing my hand closer, all fine, but to turn the screen off it is a headache cause the power button is not easily accessible via a waterproof box especially with gloves.

Working scenario for me: during navigation screen ON, bring a hand to a smartphone, screen turning OFF, bringing a hand, screen turning ON. So basically want to be able to turn on/off a screen bringing a hand.

**Comment by JonnyTech** [comment]
I use the open source "proximity service" app for that feature: https://play.google.com/store/apps/details?id=ss.proximityservice

**Comment by EvgenyZaretskiy** [comment]
Thanks I'll try.

**Comment by EvgenyZaretskiy** [comment]
> I use the open source "proximity service" app for that feature: https://play.google.com/store/apps/details?id=ss.proximityservice

Nope, it is not a working solution. App just turns off the screen and keeps it off while hand next to a smartphone. I'm talking about the ability to change a state ON/OFF of the screen using the proximity sensor.

**Comment by sonora** [comment]
Menu button / Configure profile / Global settings / Screen control: Activate Option 'Proximity sensor'.

**Comment by EvgenyZaretskiy** [comment]
> Menu button / Configure profile / Global settings / Screen control: Activate Option 'Proximity sensor'.

Activated. As I wrote I CAN turn ON the screen using a proximity sensor but wanna be able to change the state of the screen with a proximity sensor.

State - Action - State

Screen off - Hand to the phone - Screen on
Screen on - Hand to the phone - Screen off

**Comment by JonnyTech** [comment]
https://play.google.com/store/apps/details?id=com.jarsilio.android.waveup

> WaveUp is an app that wakes up your phone - switches the screen on - when you wave over the proximity sensor.
> It also locks your phone and turns off the screen if you cover the proximity sensor for one second (or a specified time)

**Comment by EvgenyZaretskiy** [comment]
> https://play.google.com/store/apps/details?id=com.jarsilio.android.waveup

Thanks, this is totally working solution for my scenario. WaveUp for On/Off and disabled proximity sensor support in Osmand. Will check on the road.

BTW, it will be good to have native OSMAND option for this.

**Comment by sjvudp** [comment]
Just a random note: Ape@map (https://play.google.com/store/apps/details?id=wmdev.apps.navBrowser) has an *optional* feature that turns on the screen for a while when you whistle... But has to use the microphone, obviously.
